.github/workflows/linkedin_release_actions.yml

```diff
@@ -0,0 +1,19 @@
+name: GitHub Actions LinkedIn
+run-name: GitHub Actions for Linkedin posts
+on:
```

Comment:
it would be cool if this workflow ran on release instead of a push to the `main` branch (which is `master` here). See: https://github.com/intelowlproject/IntelOwl/blob/master/.github/workflows/twitter_publish.yml#L2

Then you can keep
```
  push:
    branches:
      - addLinkedinRelease
```
for your tests